float64 steeringAngle
float64 SASsteeringAngle
float64 beta1
float64 tachoVelocity
float64 tachoVelocityOld
float64 leftMotorVelocity
float64 rightMotorVelocity
float64 leftMotorTorque
float64 rightMotorTorque
